lasagne/objectives.py

```diff
@@ -86,6 +86,7 @@
     "aggregate",
     "binary_hinge_loss",
     "multiclass_hinge_loss",
+    "huber_loss",
```

Comment:
And to the `.rst` file as I mentioned in https://github.com/Lasagne/Lasagne/pull/819#issuecomment-291599153. Thank you!